zone/client.h

```diff
@@ -170,6 +170,14 @@ struct XTarget_Struct
 	char Name[65];
 };
 
+typedef enum {
```

Comment:
I think an `enum class` would probably work better here, we can always safely cast in places need be